<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matchview</class>
 <widget class="QMainWindow" name="matchview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #161A22;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>1851</width>
      <height>971</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22252D;
border-radius:40px;
border: 1px solid #FFFFFF; /* Subtle blue-gray border */
</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Competition</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>Match</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/matchimg/Vector.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>620</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Team</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>28</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/compimg/Group 2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/compimg/logo football.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>28</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/compimg/Group 1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>28</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/matchimg/Vectorora.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>510</y>
      <width>15</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#line_2 {
    background-color: #D8A353; /* Match the exact background color */
    border: 2px solid #2D3748; /* Darker border to contrast with background */
    border-radius: 3px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>1260</width>
      <height>399</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/matchimg/DESIGN2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>920</y>
      <width>28</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/logout.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>920</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>790</y>
      <width>135</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #393C44;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>850</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addmatch">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>280</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HP Simplified Jpan</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#addmatch {
border: 2px solid orange; /* Subtle white border */
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(255, 184, 19);
}
QPushButton#addmatch:hover {
    background-color: #ebebeb; /* Lighter shade on hover */

}

QPushButton#addmatch:pressed {
    background-color: #ffe9c6; /* Darker shade when pressed */

}</string>
    </property>
    <property name="text">
     <string>HERE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>180</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Speedo</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>ADD A MATCH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Nova</family>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>by filling this form</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>164</y>
      <width>389</width>
      <height>335</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/matchimg/DESIGN3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="chartbutton">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>220</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Style for the pushButton_6 */
QPushButton#chartbutton {
    background-color: #B4B4B4; /* Original background color */
    color: #1A2633; /* Dark blue text for contrast */
    border: 1px solid white; /* Subtle white border */
    border-radius: 10px;
    min-width: 42px;
    min-height: 42px;
  
}

QPushButton#chartbutton:hover {
    background-color: #C5C5C5; /* Lighter shade on hover */
    min-width: 48px; /* Slight size increase */
    min-height: 48px;
}

QPushButton#chartbutton:pressed {
    background-color: #A3A3A3; /* Darker shade when pressed */
    min-width: 42px; /* Original size */
    min-height: 42px;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/matchimg/chart.png</normaloff>:/matchimg/chart.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="displaymatch">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>200</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Style for the pushButton_4 */
QPushButton#displaymatch {
    background-color: #F5A623; /* Original background color */
    color: #1A2633; /* Dark blue text */
    border: 1px solid white; /* Subtle white border */
    border-radius: 20px;
    min-width: 62px;
    min-height: 62px;
    
}

QPushButton#displaymatch:hover {
    background-color: #ffdba2; /* Lighter shade on hover */
    min-width: 70px; /* Slight size increase */
    min-height: 70px;
    
}

QPushButton#displaymatch:pressed {
    background-color: #d68f1e; /* Darker shade when pressed */
    min-width: 62px; /* Original size */
    min-height: 62px;
   
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/compimg/pencil.png</normaloff>:/compimg/pencil.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="statsbutton">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>210</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Style for the pushButton_5 */
QPushButton#statsbutton {
    background-color: #393C44; /* Original background color */
    color: #FFFFFF; /* White text for contrast */
    border: 1px solid #2c2f35; /* Subtle white border */
    border-radius: 15px;
    min-width: 52px;
    min-height: 52px;
   
}

QPushButton#statsbutton:hover {
    background-color: #4A4E58; /* Lighter shade on hover */
    min-width: 60px; /* Slight size increase */
    min-height: 60px;
    
}

QPushButton#statsbutton:pressed {
    background-color: #2D3037; /* Darker shade when pressed */
    min-width: 52px; /* Original size */
    min-height: 52px;
    
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/matchimg/statsIcon.png</normaloff>:/matchimg/statsIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>27</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>520</y>
      <width>1721</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2B2F38;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>560</y>
      <width>1161</width>
      <height>432</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/matchimg/DESIGN1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>584</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D2A9A9;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="watchmatch">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>584</y>
      <width>105</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color:black;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>Watch Match</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewallmatches">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>960</y>
      <width>208</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D8A353;
color:white;
border-radius:20px;</string>
    </property>
    <property name="text">
     <string>View All Matches</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>280</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>21</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Edit  |  Stats |  Chart</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>320</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Nova</family>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string> You can download Match details as a PDF.</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentUserPhotoLabel">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>30</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1E1F2D;
border: 1px solid #3A3D4A;
border-radius: 	30px;
box-shadow: 0 4px 10px rgba(0, 0, 0, 0.2);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_9">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>User name</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_8">
    <property name="geometry">
     <rect>
      <x>1710</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Role</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>1870</x>
      <y>60</y>
      <width>16</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/Vectorrrr.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="userbutton">
    <property name="geometry">
     <rect>
      <x>1870</x>
      <y>50</y>
      <width>21</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="compButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="playerButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="teamButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>900</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>dashboard_7</zorder>
   <zorder>dashboard_3</zorder>
   <zorder>label_5</zorder>
   <zorder>dashboard_2</zorder>
   <zorder>dashboard_4</zorder>
   <zorder>label_6</zorder>
   <zorder>label</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>line_2</zorder>
   <zorder>label_20</zorder>
   <zorder>dashboard_6</zorder>
   <zorder>line</zorder>
   <zorder>dashboard_5</zorder>
   <zorder>addmatch</zorder>
   <zorder>label_18</zorder>
   <zorder>label_21</zorder>
   <zorder>chartbutton</zorder>
   <zorder>displaymatch</zorder>
   <zorder>statsbutton</zorder>
   <zorder>label_15</zorder>
   <zorder>label_16</zorder>
   <zorder>label_17</zorder>
   <zorder>watchmatch</zorder>
   <zorder>viewallmatches</zorder>
   <zorder>label_22</zorder>
   <zorder>label_23</zorder>
   <zorder>currentUserPhotoLabel</zorder>
   <zorder>dashboard_9</zorder>
   <zorder>dashboard_8</zorder>
   <zorder>label_24</zorder>
   <zorder>userbutton</zorder>
   <zorder>compButton</zorder>
   <zorder>playerButton</zorder>
   <zorder>teamButton</zorder>
   <zorder>logout</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
